docs/faq.md

```diff
@@ -0,0 +1,33 @@
+# General FAQ
+
+## AMD GPUs crashing the entire system
+
+Changing resolution, using DRI_PRIME and doing various other things can cause crashes, but running the command `echo high| sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level` or adding the `amdgpu.dpm=0` to the kernel commandline stops these crashes.
+
+## The MacBook's screen freezes, fans run up, and then it turns off
+
+A broken runtime call crashed Linux; try a patched kernel (or updating if you're already using it), or boot macOS and check the crashlog. If you have `apple-bce`, try updating it to 0.2. You may need a custom live ISO. You can also try adding the ``efi=noruntime`` option to GRUB's command line paramaters, or adding ``efivarfs /sys/firmware/efi/efivars efivarfs ro,remount,nofail 0 0`` to the end of your /etc/fstab file (although don't use both at the same time).
+
+## WiFi is not working
+
+Follow the [wifi guide](https://wiki.t2linux.org/guides/wifi). If you followed that, try updating, or using a different ISO. You could also try reloading brcmfmac: ``sudo modprobe -r brcmfmac && sudo modprobe brcmfmac``.
```

Comment:
typo. Wi-Fi not wifi

Reply:
I wouldn't call this a typo. All it's missing is it's "-". People will get the point if it's spelled like "wifi" or "Wi-Fi" so there's no point in changing it.

Reply:
I just meant for aesthetic purposes

Reply:
Then it's not really a typo now is it?

Reply:
Changed the comment

Reply:
Well if I change this then I would have to change it everywhere...

Reply:
Currently we are inconsistent with all of "WiFi", "wifi" and "Wi-Fi" being used in different places. This PR is getting quite big which means it is taking a while to get it merged so if we want to change them all to be the same "Wi-Fi", that can be done in a separate PR.

```sh
$ grep -r Wi-Fi docs | wc -l
3
$ grep -r WiFi docs | wc -l
8
$ grep -r wifi docs | wc -l # note that some of these are in code blocks or links
33
```